File: src/entity/ParkingLot.java
package entity;

import java.util.List;

public class ParkingLot {
    private int space;
    private List<Vehicle> vehicles;

    public int getSpace() {
        return space;
    }

    public void setSpace(int space) {
        this.space = space;
    }

    public List<Vehicle> getVehicles() {
        return vehicles;
    }

    public void setVehicles(List<Vehicle> vehicles) {
        this.vehicles = vehicles;
    }

    public ParkingLot() {
    }

    public ParkingLot(int space, List<Vehicle> vehicles) {
        this.space = space;
        this.vehicles = vehicles;
    }

    @Override
    public String toString() {
        return "ParkingLot{" +
                "space=" + space +
                ", vehicles=" + vehicles +
                '}';
    }

}

File: dotnet-version/ParkingSystemDotnet/Models/ParkingLot.cs
using System.Collections.Generic;
using ParkingSystemDotnet.Models;

namespace ParkingSystemDotnet.Models
{
    public class ParkingLot
    {
        public int Space { get; private set; }
        public List<Vehicle> Vehicles { get; private set; }

        public ParkingLot(int space, List<Vehicle> vehicles)
        {
            Space = space;
            Vehicles = vehicles;
        }
    }
}